Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -197,6 +202,20 @@ void UpdateLineHeight()
 				_view.SetLineSpacing(0, (float)Element.LineHeight);
 		}
 
+		void UpdateMaxLines()
+		{
+			if (Element.MaxLines > 0)
+			{
+				Control.SetSingleLine(Element.MaxLines == 1);
+				Control.SetMaxLines(Element.MaxLines);
+			}
+			else
+			{
+				Control.SetSingleLine(true);
+				Control.SetMaxLines(1);
```

Comment:
I'm might be wrong here, but I'm not sure the default is single line labels.

Reply:
Correct, will change